src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx

```diff
@@ -49,79 +42,106 @@ export default function (props: BloodPressureVisualizationProps) {
     const _maxSystolic = 250;
     const yInterval: ClosedInterval = { values: [_minDiastolic, _maxSystolic] };
     const axis: Axis = { yRange: yInterval, yIncrement: 50, xIncrement: (80 / 7) };
-    const [bloodPressureData, setDataForGraph] = useState<Map<string, BloodPressureDumbbell> | undefined>(undefined);
+    const [bloodPressureData, setData] = useState<Map<string, BloodPressureDataPoint[]> | undefined>(undefined);
```

Comment:
For consistency, could this be either:
```
const [data, setData] ...
```
or
```
const [bloodPressureData, setBloodPressureData] ...
```

Reply:
Updated